src/main/groovy/org/broadinstitute/orsp/ConsentCollectionLinkDTO.groovy

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.orsp
+import org.broadinstitute.orsp.IssueType
+
+class ConsentCollectionLinkDTO {
+
+    BigInteger id
```

Comment:
Ah ... here I think this should be a Long to match how it is defined in `StorageDocument`

Reply:
If we put Long, we will get an error, can not be casted, this is why we defined it as BigInteger

Reply:
I understand that you're seeing a cast error, but that error should be **because** this is not a Long. My point is that the cast is only fixing a symptom of the real problem, and I think this is the source of the real problem.